Gruntfile.js

```diff
@@ -1,10 +1,10 @@
-module.exports = function(grunt) {
+module.exports = function (grunt) {
     'use strict';
     // Project configuration.
-    var saucekey = process.env.saucekey;
+    var saucekey = process.env.SAUCE_ACCESS_KEY || process.env.saucekey;

```

Comment:
I'll remove the settings from the `yml` file so this won't be needed, can you revert this particular change (and the one on line 7)?